spec.markdown

```diff
@@ -348,19 +348,11 @@ only deliver integrity metadata on a [potentially secure origin][].  See
 
 The following algorithm details these restrictions:
 
-1.  Let <var>request</var> be the request that fetched
-    <var>resource</var>.
-2.  If the [response type][] is `opaque`, return `false`.
-3.  If the [mode][fetch-mode] of <var>request</var> is `CORS`,
-    return `true`.
-4.  If the [origin][fetch-origin] of <var>request</var> is
-    <var>resource</var>'s origin, return `true`.
-5.  Return `false`.
-
-Step 3 returns `true` if the fetch was a CORS-enabled request. If the
-fetch failed the CORS checks, it won't be available to us for integrity
-checking because it won't have loaded successfully.
-{:.note}
+1.  Let <var>response</var> be the response that results from
+    fetching the <var>resource</var>.
```

Comment:
Should "fetching" be replaced with a link to the Fetch algorithm? Right now it's ambiguous, especially because I think it's referring to a formal definition.